app/process/shared/HighwayClasses.lua

```diff
@@ -1,17 +1,29 @@
 -- https://wiki.openstreetmap.org/wiki/DE:Key:highway
 -- https://wiki.openstreetmap.org/wiki/Attribuierung_von_Stra%C3%9Fen_in_Deutschland
+-- We keep the different highway classes separate so we can use them for filtering
+-- to combine them use the function JoinSets in the ~/process/helper/JoinSets
+
+-- "*_link" bedeutet "Autobahnzubringer", "Anschlussstelle", "Auf- / Abfahrt"
 HighwayClasses = Set({
```

Comment:
@tordans please have a look at the new categories. In that way there where handy to use in the rest of the code